REUSE.toml

```diff
@@ -0,0 +1,126 @@
+# SPDX-License-Identifier: curl
+# SPDX-FileCopyrightText: Daniel Stenberg, <daniel@haxx.se>, et al.
+
+version = 1
+SPDX-PackageName = "curl"
+SPDX-PackageSupplier = "Daniel Stenberg <daniel@haxx.se>"
+SPDX-PackageDownloadLocation = "https://curl.se"
```

Comment:
```suggestion
SPDX-PackageDownloadLocation = "https://curl.se/"
```
minor nit

Reply:
Fine with me, it was just copied from 

https://github.com/curl/curl/blob/2abfc759b98852cd916ebfb6207d296762b1171a/.reuse/dep5#L4

Reply:
Fixed